scripts/mobilecomponents.sh

```diff
@@ -62,14 +62,6 @@ sed -i -e "s/visible: root.rightAction/visible: root.rightAction \&\& \!Qt.input
 # https://bugs.kde.org/show_bug.cgi?id=394204
 sed -i -e "s/width: backgroundRect/enabled: root.enabled;    width: backgroundRect/g" src/controls/templates/private/PassiveNotification.qml
 
-# another hack. Do not include the Kirigami resources (on static build). It causes
-# double defined symbols in our setting. I would like a nicer fix for this
-# issue, but failed to find one. For example, not adding the resource in
-# our build causes the qrc file not to be generated. Manual generation
-# of the resource file (using rcc) introduces the double symbols again. 
-# so it seems some Kirigami weirdness (but their staticcmake example compiles
-# correctly).
-sed -i -e "s/#include <qrc_kirigami.cpp>/\/\/#include <qrc_kirigami.cpp>/" src/kirigamiplugin.cpp
```

Comment:
i did this today and it was needed otherwise i was getting weird duplicate symbols when building.

Reply:
the travis build failed and it kind of confirms my observation:
https://api.travis-ci.org/v3/job/384204511/log.txt
```

/workspace/ndk-arm/bin/../lib/gcc/arm-linux-androideabi/4.9.x/../../../../arm-linux-androideabi/bin/ld: error: CMakeFiles/subsurface-mobile.dir/qrc_kirigami.cpp.o: multiple definition of 'qInitResources_kirigami()'

/workspace/ndk-arm/bin/../lib/gcc/arm-linux-androideabi/4.9.x/../../../../arm-linux-androideabi/bin/ld: CMakeFiles/subsurface-mobile.dir/mobile-widgets/qml/kirigami/src/kirigamiplugin.cpp.o: previous definition here

/workspace/ndk-arm/bin/../lib/gcc/arm-linux-androideabi/4.9.x/../../../../arm-linux-androideabi/bin/ld: error: CMakeFiles/subsurface-mobile.dir/qrc_kirigami.cpp.o: multiple definition of 'qCleanupResources_kirigami()'

/workspace/ndk-arm/bin/../lib/gcc/arm-linux-androideabi/4.9.x/../../../../arm-linux-androideabi/bin/ld: CMakeFiles/subsurface-mobile.dir/mobile-widgets/qml/kirigami/src/kirigamiplugin.cpp.o: previous definition here

collect2: error: ld returned 1 exit status
```

Reply:
It was not a problem for iOS, but if it breaks android or mobile on desktop I'll leave it.

I just removed this commit.